src/couch_mrview_http.erl

```diff
@@ -424,6 +424,16 @@ row_to_json(Id0, Row) ->
     ?JSON_ENCODE(Obj).
 
 
+check_compatible(K1, K2, Props) ->
+    case {lists:keymember(K1, 1, Props), lists:keymember(K2, 1, Props)} of
+        {true, true} ->
+            Err = ["`", K1, "` and `", K2,"` are incompatible"],
```

Comment:
This is an unusual way to compose a string... Any reason not to use `io_lib:format("~p and ~p are incompatible", [K1, K2])` here?